components/merkledb/src/tests.rs

```diff
@@ -32,3 +33,67 @@ fn should_compile() {
     let _: SparseListIndex<_, ()> = SparseListIndex::new_in_family("", "", &fork);
     let _: ValueSetIndex<_, ()> = ValueSetIndex::new_in_family("", "", &fork);
 }
+
+#[test]
+fn checkpoints() {
```

Comment:
I wonder if it's possible to reliably generate enough data for the DB in order for it to dump some of it into SST files. As far as I understand, checkpoints hard-link SST files and copy the newer logs, so it could be interesting to check that these both mechanisms work as expected.

Reply:
> I wonder if it's possible to reliably generate enough data for the DB in order for it to dump some of it into SST files.

I believe closing and reopening a database should flush memtable into sst.

Reply:
Potentially, the method `merge_sync()` should flush memtables into sst files as well.